squirrel/sqoptimizer.cpp

```diff
@@ -8,6 +8,7 @@
 #include "sqoptimizer.h"
 
 SQOptimizer::SQOptimizer(SQFuncState & func_state) : fs(&func_state), jumps(func_state._lineinfos._alloc_ctx), codeChanged(false) {}
+#undef _DEBUG_DUMP
```

Comment:
Need to eventually add dump support to optimizer, even if later in a separate commit